composer.json

```diff
@@ -14,7 +14,7 @@
         "php": ">=8.0",
         "doctrine/common": "^3.1",
         "doctrine/persistence": "^2.2",
-        "doctrine/dbal": "^2.13",
+        "doctrine/dbal": "^3.1",
```

Comment:
Would it be possible to use `^2.13 || ^3.1`?

Reply:
Sure.
I have used single | to be consistent.